recordedit/form.controller.js

```diff
@@ -114,12 +115,6 @@
             for (var i = 0; i < $rootScope.reference.columns.length; i++) {
                 var col = $rootScope.reference.columns[i];
 
-                // If this column is disabled, its value is already in the format
-                // needed for submission.
-                if (col.getInputDisabled(context.appContext)) {
-                    continue;
-                }
-
                 var rowVal = row[col.name];
                 if (rowVal) {
```

Comment:
This is not related to what you changed in this PR. But the [if statement in `default`](https://github.com/informatics-isi-edu/chaise/blob/261a5780b623466ac1163264d4c28271e0b46018/recordedit/form.controller.js#L136) will never be true, since we have this if statement here.